src/main/java/com/mycompany/SpringBootStarter.java

```diff
@@ -0,0 +1,15 @@
+package com.mycompany;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.web.servlet.ServletComponentScan;
+
+@SpringBootApplication
+@ServletComponentScan
```

Comment:
Why we need @ServletComponentScan over @ComponentScan?

Reply:
It should be @ComponentScan. Now I have read that @SpringBootApplication already includes @ComponentScan, I'll fix it

Reply:
I think we shouldn't put @ before annotation in comments, it's user-mention pattern for github :)